pkg/genericapiserver/genericapiserver.go

```diff
@@ -892,6 +894,25 @@ func (s *GenericAPIServer) InstallSwaggerAPI() {
 		},
 	}
 	swagger.RegisterSwaggerService(swaggerConfig, s.HandlerContainer)
+	openAPIConfig := openapi.Config{
+		SwaggerConfig:  &swaggerConfig,
+		IgnorePrefixes: []string{"/swaggerapi"},
+		Info: &spec.Info{
+			InfoProps: spec.InfoProps{
+				Title:   "Kubernetes APIs",
+				Version: "1.4",
```

Comment:
We wont be able to keep it up to date.
I am fine with just removing it for now.